### PR TITLE
Integrate OPS Schedule issue generation inside DAOBot

### PR DESCRIPTION
**Pull request by AnaLoznianu**
Fixes https://app.zenhub.com/workspaces/oceandao-60a669231c2ed7000e90684b/issues/oceanprotocol/oceandao/35

Integration of the OPS Script Issues generation inside DAOBot

Test requirements:
- Inside the Funding Rounds table, add a new checkbox field called 'OPS'. This field is updated by the script once it generates the tasks for a round. In this way, we are making sure the issues are not generated multiple times
- In the .env please add the following variables:
# Build type 
BUILD_TYPE = 'DEV'. To run and test the issue generation set the BUILD_TYPE='PROD'

# OPS Schedule GitHub variables
GITHUB_REPOSITORY='Repository'
GITHUB_ORGANISATION='Organisation'
GITHUB_TOKEN='Your Github token'

### CODE REVIEW

File: src/utils/ops/repeatable_tasks.json [hunk 1]
```diff
@@ -0,0 +1,140 @@
+[
```
Comment (AnaLoznianu) [inline src/utils/ops/repeatable_tasks.json:1]: @idiom-bytes Please add your OPS tasks suggestions here or feel free to commit new tasks to this repo

**Comment by AnaLoznianu** [comment]
@trentmc @alexcos20 @idiom-bytes This is a script that generates repeatable items on GitHub for each round we are adding inside Airtable. Feel free to add new repeatable items for your working group's activities.

**Comment by idiom-bytes** [comment]
Hi, I have updated the codebase so it's looking at env.BUILD_TYPE, and updated some task descriptions to improve them.
- I believe this PR looks good. I was able to run locally with the `OPS` column.
- I have updated oDAO-Prod to have a row for R13.

File: src/scripts/ops/generate_issues.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+const roundRawIssues = require('../../utils/ops/repeatable_tasks.json')
+const {
+    getRoundsSelectQuery,
+    updateRoundRecord
+  } = require('../../airtable/airtable_utils')
+
+const organisation = process.env.GITHUB_ORGANISATION || 'oceanprotocol'
+const repo = process.env.GITHUB_REPOSITORY || 'oceandao'
+const token = process.env.GITHUB_TOKEN
+const buildType = process.eventNames.BUILD_TYPE || 'DEV'
```
Comment (idiom-bytes) [inline src/scripts/ops/generate_issues.js]: I have updated this to be `process.env.BUILD_TYPE`

**Review by idiom-bytes — APPROVED** [review]
Changes look good. I especially like the integration of the oDAO configuration as args, so the tickets communicate the right dates & information.

Some of the links that the scripts generate are wrong (for an example, the links to Port), and these should eventually get addressed.

I vote this PR gets submitted, and we continue to iterate.

File: src/scripts/ops/generate_issues.js (hunk 2 above)
Comment (AnaLoznianu) [reply src/scripts/ops/generate_issues.js]: Great catch, I missed it

**Comment by AnaLoznianu** [comment]
I'll integrate this PR after the voting period ends for Round 12.

**Pull request merged**